pkg/vsphere/simulator/model.go

```diff
@@ -125,11 +130,25 @@ func (m *Model) Create() error {
 		host := object.NewHostSystem(client, info.Result.(types.ManagedObjectReference))
 		hosts = append(hosts, host)
 
+		if dvs != nil {
+			config := &types.DVSConfigSpec{
+				Host: []types.DistributedVirtualSwitchHostMemberConfigSpec{{
+					Operation: string(types.ConfigSpecOperationAdd),
+					Host:      host.Reference(),
+				}},
+			}
+
+			_, _ = dvs.Reconfigure(ctx, config)
```

Comment:
That's for silencing go-errcheck when intentionally ignoring an error.

![flycheck](https://cloud.githubusercontent.com/assets/30171/22261334/601a2628-e221-11e6-8393-1e9e54c42a41.png)